js/apps/admin-ui/src/components/key-value-form/ValueSelect.tsx

```diff
@@ -41,6 +41,11 @@ export const ValueSelect = ({
       ))}
     </Select>
   ) : (
-    <KeycloakTextInput id="customValue" data-testid={rest.name} {...field} />
+    <KeycloakTextInput
+      id={t("customValue")}
```

Comment:
The `id` should not be translated, perhaps we can just remove this?
```suggestion
```

Reply:
yeah, I think we can remove it